google-cloud-clients/google-cloud-core/src/main/java/com/google/cloud/ByteArray.java

```diff
@@ -117,7 +117,7 @@ public final void copyTo(ByteBuffer target) {
    * @throws IndexOutOfBoundsException if the target is not large enough to hold the data
    */
   public final void copyTo(byte[] target) {
-    byteString.copyTo(target, 0, 0, length());
+    byteString.copyTo(target, 0);
```

Comment:
Are you sure that byteString#copyTo with two params is correct to fall back to?
Are there any other option?